articles/getting-started/start-workflow.md

```diff
@@ -1,6 +1,6 @@
 ---
-uid: RunningAWorkflow
-title: Running a Workflow
+uid: StartAWorkflow
+title: Start a Workflow
 ---
 
 Once all nodes have been placed and linked correctly, and all **Configuration** properties have been set, it is now possible to run a workflow. Note that some aspects of Bonsai are only available in specific contexts; for instance, the GUIs mentioned above can only be opened when a workflow is not running. Once a workflow is running, these GUIs are not accessible, but visualizers for certain nodes can be opened to view the streaming data.
```

Comment:
References stuff "above" but this is now a separate page. I didn't fix this when I separated the pages, but we should verify that this language is removed